pkg/resource/image_metadata.go

```diff
@@ -238,3 +242,15 @@ func ExtractPrometheusConfigurationFromImage(dockerImage *dockerv10.DockerImage)
 
 	return
 }
+
+// IsPersistenceEnabled verifies if the image has labels indicating that persistence is enabled
+func IsPersistenceEnabled(dockerImage *dockerv10.DockerImage) (enabled bool) {
+	if !dockerImageHasLabels(dockerImage) {
+		return false
+	}
+	var err error
+	if enabled, err = strconv.ParseBool(dockerImage.Config.Labels[LabelKeyOrgKiePersistenceRequired]); err != nil {
```

Comment:
is this label set by someone?

Reply:
Yes, by the `kogito-codegen`:
https://github.com/kiegroup/kogito-runtimes/blob/master/kogito-codegen/src/main/java/org/kie/kogito/codegen/metadata/PersistenceLabeler.java#L30